rust/hyperlane-base/src/contract_sync/mod.rs

```diff
@@ -58,29 +59,32 @@ where
             .with_label_values(&[label, chain_name]);
 
         loop {
-            let Ok((from, to, eta)) = cursor.next_range().await else { continue };
-            debug!(from, to, "Looking for for events in block range");
+            indexed_height.set(cursor.latest_block() as i64);
+            let Ok((action, eta)) = cursor.next_action().await else { continue };
```

Comment:
would it be useful to warn log the errors?

Reply:
Yes probably, but I'd prefer not to make more changes outside the scope of this PR, since I was hoping not to do any changes at all